### PR TITLE
Created head

### PR DESCRIPTION
**Pull request by LazarevSlava**
Created Header folder
Changed App.jsx for using header instead Search and InfoPanel. Now i use them in head

# Changed

- A new component ToggleTodo was added

## Checklist

- [ ] Linter checks
- [ ] Code style checks

### CODE REVIEW

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import style from './header.module.scss';
+import InfoPanel from '../InfoPanel/InfoPanel';
+import Search from '../search_form/Search';
+
+function Header({ onSearch, isLoading, ip, location, timezone, isp }) {
+  return (
+    <div className={style.finder}>
+      <h1 className={style.title}> IP Address Tracker</h1>
```
Comment (michey85) [inline src/components/header/Header.jsx:8]: Why did you add a space inside h1 before text?

Comment (LazarevSlava) [reply src/components/header/Header.jsx:8]: oops

File: src/App.jsx [hunk 2]
```diff
@@ -12,10 +10,14 @@ function App() {
...
     <div>
-      <div className="header">
-        <Search onSearch={handleSearch} />
-        <InfoPanel ip="mmm" location="mmm" timezone="mmm" isp="mmm" isLoading={isLoading} />
-      </div>
+      <Header
+        ip="mmm"
+        location="mmm"
+        timezone="mmm"
+        isp="mmm"
```
[the remark this answers is not in the document]
Comment (LazarevSlava) [reply src/App.jsx:17]: ok

File: src/components/button/button.module.scss [hunk 3]
```diff
@@ -9,3 +9,9 @@
 .request:hover {
   background-color: maroon;
 }
+@media (max-width: 475px) {
+  .request {
+    border-radius: 0 0 10px 10px;
+    width: 80%;
```
[the remark this answers is not in the document]
Comment (LazarevSlava) [reply src/components/button/button.module.scss:15]: 80% because it looks good in mobil with input form

[the remark this answers is not in the document]
Comment (LazarevSlava) [reply src/components/button/button.module.scss:15]: better then 
Снимок экрана 2023-12-08 в 13 37 05

[the remark this answers is not in the document]
Comment (michey85) [reply src/components/button/button.module.scss:12]: Oh my god, man. I missed that you used wrong approach. Never! Do you hear it? Never use max-width approach. Or you will look like a dummy beginner. It always has to be mobile first 💡

**Comment by LazarevSlava** [comment]
Can I marge it?

File: src/components/utils/_vars.scss [hunk 4]
```diff
@@ -5,6 +5,7 @@ $blue: blue;
 $mainFont: 'Playfair Display', serif;
 
 //screens
+$mobil: 'max-width: 475px';
```
Comment (michey85) [inline src/components/utils/_vars.scss:8]: It has to be different. Both - with naming and with a value.
First of all `mobile` not `mobil`. And usually mobile is something by default. Yes, we might need to have different shapes of phones at times, thin we can try a different naming. Like `mediumPhone`, `largePhone` or `xPhone`, `xlPhone`.
Secondly, we discussed this only this morning, you have to avoid using `max-width`. We can proceed with this bullshit for now, but for the next project you'll get a second yellow card for this 🟨

**Review by michey85 — APPROVED** [review]

File: src/components/utils/_vars.scss (hunk 4 above)
Comment (LazarevSlava) [reply src/components/utils/_vars.scss:8]: I try to find a positive thing in your message. And now I see that we are talking about our next project, and that means I steel have a chance to become a programmer :)
